rpm/gratia-probe.spec

```diff
@@ -400,7 +436,7 @@ Requires: htcondor-ce
 License: See LICENSE.
 
 %description htcondor-ce
-The HTCondor-CE probe for the Gratia OSG accounting system.
+The HTCondor-CE probe for the Gratia OSG accounting system. Designed for use with non-HTCondor batch systems.
```

Comment:
This suggests we don't want it running if the batch system is condor?

Reply:
Yeah, config and crons would fight

Reply:
wanna add a Conflict?

Reply:
Done in the latest commit